Tasks/AzureRmWebAppDeployment/AzureRmWebAppDeployment.ts

```diff
@@ -18,8 +18,8 @@ async function run() {
 		var deployToSlotFlag: boolean = tl.getBoolInput('DeployToSlotFlag');
 		var resourceGroupName: string = tl.getInput('ResourceGroupName');
 		var slotName: string = tl.getInput('SlotName');
-		var package: string = tl.getPathInput('Package');
-		var setParametersFile: string = tl.getPathInput('SetParametersFile');
+		var webDeployPkg: string = trimDoubleQuotes(tl.getPathInput('Package'));
```

Comment:
If you have to do trim for some reason do it only on windows.  I realize there might be powershell tasks written in a bad way that required users to double quote a single value if it had spaces (bad but OK).  If that's the case, in the trim function condition on windows (task lib has function).

Another better option in my opinion would be to go into task lib and in getPathInput function, if windows trim double quotes.

Which brings to next point.  Ensure you're using task-lib 0.9.x.  Add package.json so build pulls in explicit task lib version.